src/Aspire.Dashboard/wwwroot/js/app.js

```diff
@@ -70,7 +70,7 @@ window.copyTextToClipboard = function (id, text, precopy, postcopy) {
             copyIcon.style.display = 'none';
             checkmarkIcon.style.display = 'inline';
         })
-        .catch(() => {
+        .catch((e) => {
```

Comment:
What is this for? You could write it to the console.log if it's not already written.